### PR TITLE
Docs: Add evaluation script.

### PR DESCRIPTION
**Pull request by taranjeet**
@deshraj

### CODE REVIEW

File: docs/source/evaluation_scripts.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+The logic for evaluating and judging a submission is custom and varies from challenge to challenge. But the overall structure of evaluation script is fixed due to architectural reasons.
+
+It is mandatory for evaluation script to write a `evaluate` function. This is kind of a main function, which is used by worker to evaluate any submission message.
+
+The syntax of evaluate function is
+
+```
+
+def evaluate(test_annotation_file, user_annotation_file, phase_name):
```
Comment (deshraj) [inline docs/source/evaluation_scripts.md]: Replace `phase_name` with `phase_codename`

Comment (taranjeet) [inline docs/source/evaluation_scripts.md]: updated.

Comment (deshraj) [inline docs/source/evaluation_scripts.md]: Thanks. :)

**Comment by deshraj** [comment]
This looks good as an initial start. I will add more to this soon to complete it. Thanks @taranjeet for taking a lead on this. :)

**Pull request merged**